Scenes/Levels/TemplateLevel/TemplateLevel.gd

```diff
@@ -30,7 +30,7 @@ func place_tower(position, tower):
 func spawn_enemy(position):
 	var new_enemy = enemy.instance()
 	new_enemy.position = position
-	new_enemy.debug_mode = true
+	new_enemy.debug_mode = false
```

Comment:
This is the default so we can just delete this

Reply:
oh, shit sorry I actually meant to take this out I just turned it off for myself. do we want to leave it off?